cli/pkg/kctrl/cmd/package/release/release.go

```diff
@@ -134,7 +134,7 @@ func (o *ReleaseOptions) Run() error {
 		case release.Resource != nil:
 			err = o.releaseResources(appSpec, *pkgBuild, &pkgConfigs.Pkgs[0], &pkgConfigs.PkgMetadatas[0])
 			if err != nil {
-				return nil
+				return err
 			}
```

Comment:
nit: We can just do a return directly
```go
return o.releaseResources(appSpec, *pkgBuild, &pkgConfigs.Pkgs[0], &pkgConfigs.PkgMetadatas[0])
```

Reply:
It's not the last call in the Run(), so proposed change will skip o.printNextSteps() below.

Reply:
@goshkis We shouldn't be printing the next steps if `error` is encountered. 
Apart from this change, LGTM